net.zdechov.c_evo.yaml

```diff
@@ -0,0 +1,60 @@
+app-id: net.zdechov.c_evo
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.freepascal
+command: c-evo
+rename-desktop-file: c-evo.desktop
+rename-icon: c-evo
+finish-args:
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=ipc
+  - --device=dri
+
+modules:
+  - shared-modules/gtk2/gtk2.json
+
+  - name: c-evo
+    buildsystem: simple
+    build-commands:
+      - |
+        . /usr/lib/sdk/freepascal/enable.sh
+        cd tags/1.3.6/
+        lazbuild Integrated.lpr
+    post-install:
+      - mkdir -p $FLATPAK_DEST/share/c-evo
+      - cp -r tags/1.3.6/AI $FLATPAK_DEST/share/c-evo
```

Comment:
1.3.6 is going to break on every new tag

Reply:
see also https://github.com/flathub/flathub/pull/5833/files#r1836745009

Reply:
this could also have been `subdir` but this should be necessary with what I referenced above.